packages/foreman/rubygem-ipaddress/rubygem-ipaddress.spec

```diff
@@ -47,10 +46,8 @@ Documentation for %{pkg_name}
 # Remove un-needed file
 # See https://github.com/bluemonk/ipaddress/issues/23
 rm .%{gem_instdir}/.document
-
-pushd .%{gem_instdir}
-%patch0 -p1
-popd
+rm .%{gem_instdir}/.gitignore
+rm .%{gem_instdir}/.rock.yml
```

Comment:
you can put those via `%exclude` in the `%files` section, which I'd prefer (but not block on)